src/hdmf/common/table.py

```diff
@@ -122,27 +125,22 @@ def __getitem_helper(self, arg, **kwargs):
         Internal helper function used by __getitem__ to retrieve a data value from self.target
 
         :param arg: Integer index into this VectorIndex indicating the element we want to retrieve from the target
-        :param kwargs: keyword arguments to pass into *self.target.get*
+        :param kwargs: any additional arguments to *get* method of the self.target VectorData
+        :return: Scalar or list of values retrieved
         """
         start = 0 if arg == 0 else self.data[arg-1]
         end = self.data[arg]
         return self.target.get(slice(start, end), **kwargs)
 
     def __getitem__(self, arg):
-        """
-        Select elements in this VectorIndex and retrieve the corrsponding data from the self.target VectorData
-
-        :param arg: slice or integer index indicating the elements we want to select in this VectorIndex
-        :return: Scalar or list of values retrieved
-        """
```

Comment:
Why remove the docstring? This function is still included in the API documentation.